app/api/alibaba.ts

```
@@ -83,28 +82,77 @@ async function request(req: NextRequest) {
-      fetchOptions.body = clonedBody;
+      let jsonBody: any = {};
+
+      try {
+        jsonBody = JSON.parse(clonedBody);
+
+        // Move input.messages to messages at the root level if present
+        if (jsonBody.input && Array.isArray(jsonBody.input.messages)) {
+          jsonBody.messages = jsonBody.input.messages;
+
+          // Remove input.messages to avoid duplication
+          delete jsonBody.input;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)